src/api/spec/services/event_receivers_fetcher_spec.rb

```diff
@@ -22,7 +22,7 @@
   let!(:bs_request) { create(:bs_request_with_submit_action, state: :new, target_package: package) }
 
   describe '#call' do
-    subject { described_class.new(Event::RequestCreate.first, receiver_role).call }
+    subject { described_class.new(receiver_role).call }
```

Comment:
This change needs to be undone. An event is needed to initialize an instance of _EventReceiversFetcher_.

Reply:
Yes, I found it now :sweat_smile: